assignment/impl/src/java/org/sakaiproject/assignment/impl/AssignmentServiceImpl.java

```diff
@@ -1840,6 +1840,42 @@ public String getSubmissionStatus(String submissionId) {
         return i18nStatus;
     }
 
+    @Override
+    public String getSubmissionStatusDate(String submissionId) {
```

Comment:
This method name implies that it will return the "status date", not that it will return a differently formatted status string. It should have a better name.

Also, this method implementation duplicates a lot of code from getSubmissionStatus. The difference between the two methods should probably be factored out instead.

Reply:
I agree with you so I have implemented it in the same function.

Reply:
Thanks @plukasew